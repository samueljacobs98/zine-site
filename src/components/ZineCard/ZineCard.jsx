import "./ZineCard.scss";

const ZineCard = () => {
  return <div>ZineCard</div>;
};

export default ZineCard;
