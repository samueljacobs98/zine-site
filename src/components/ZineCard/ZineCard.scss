@use "../../styles//variables.scss" as *;
.zineCard {
  max-width: 320px;
  height: 380px;
  width: 100%;
  padding: 0.5rem;
  background-color: $primary-color;
  border-radius: 10px;
  box-shadow: rgb(0 0 0 / 35%) 13px 6px 10px;
  display: flex;
  &__leftContent {
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    align-items: center;
    width: 85px;
    &-title {
      -ms-writing-mode: tb-rl;
      -webkit-writing-mode: vertical-tb;
      writing-mode: vertical-rl;
      transform: rotate(180deg);
      font-size: $font-size-smaller;
      color: $secondary-color;
    }
    &-profileName {
      -ms-writing-mode: tb-rl;
      -webkit-writing-mode: vertical-tb;
      writing-mode: vertical-rl;
      transform: rotate(180deg);
      font-size: $font-size-tiny;
      color: $secondary-color;
    }
    &-profileImage {
      width: 50%;
    }
  }

  &__zineImage {
    width: 100%;
    height: 100%;
  }
}
