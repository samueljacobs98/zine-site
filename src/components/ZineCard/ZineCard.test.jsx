import { customRender } from "../../utils/testUtils";
import ZineCard from "./ZineCard.jsx";

it("should match ZineCard component snapshot", () => {
  const { container } = customRender(<ZineCard />);

  expect(container).toMatchSnapshot();
});
