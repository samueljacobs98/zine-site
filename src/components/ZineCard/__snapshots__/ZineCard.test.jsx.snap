// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match ZineCard component snapshot 1`] = `
<div>
  <div
    class="zineCard"
  >
    <div
      class="zineCard__leftContent"
    >
      <h2
        class="zineCard__leftContent-title"
      />
      <h3
        class="zineCard__leftContent-profileName"
      />
      <img
        alt=""
        class="zineCard__leftContent-profileImage"
      />
    </div>
    <div
      class="zineCard__rightContent"
    >
      <img
        alt=""
        class="zineCard__zineImage"
      />
    </div>
  </div>
</div>
`;
